doc/api/inspector.md

```diff
@@ -0,0 +1,80 @@
+# Inspector
+
+> Stability: 1 - Experimental
+
+The `inspector` module provides an API for interacting with the V8 inspector.
+
+It can be accessed using:
+
+```js
+const inspector = require('inspector');
+```
+
+## Class: inspector.Session
+<!-- YAML
+added: REPLACEME
+-->
+
+The `inspector.Session` class is a primary means of dispatching the messages to
+the V8 inspector backend and receiving message responses and notifications.
+
+It needs to be connected before the messages can be sent to the inspector
+backend.
+
+### Event 'inspectorNotification'
+<!-- YAML
+added: REPLACEME
+-->
+
+* {Object} The notification
+
+Emitted when a notification from an inspector backend was received.
+
+It is also possible to subscribe to a specific inspector notification type by
+specifying the method. E.g. this snippet will print the reason for application
+been suspended when the [Debugger.paused][debugger-paused] event is recieved
+be called whenever the program execution is suspended:
```

Comment:
Nit: Use the same anchor name as the text per convention.

```markdown
[`'Debugger.paused'`][]
```

here, and

```markdown
[`'Debugger.paused'`]: https://chromedevtools.github.io/devtools-protocol/v8/Debugger/#event-paused
```

----

Also I feel this paragraph needs some rewording. The following sounds better to me:

```md
It is also possible to subscribe to a specific inspector notification type.
The following snippet installs a listener on the [`'Debugger.paused'`][]
event, and prints the reason for program suspension whenever program
execution is suspended (through breakpoints, for example).
```